include/deal.II/base/vectorization.h

```diff
@@ -194,27 +196,27 @@ class VectorizedArrayBase
   }
 
   /**
-   * @return An iterator pointing to the end of the underlying data.
+   * @return An iterator pointing to the beginning of the underlying data (`const`
+   * version).
    */
-  VectorizedArrayIterator<T>
-  end()
+  VectorizedArrayIterator<const T>
+  begin() const
   {
-    return VectorizedArrayIterator<T>(static_cast<T &>(*this),
-                                      T::n_array_elements);
+    return VectorizedArrayIterator<const T>(static_cast<const T &>(*this), 0);
   }
 
   /**
-   * @return An iterator pointing to the beginning of the underlying data (const
-   * version).
+   * @return An iterator pointing to the end of the underlying data.
    */
-  VectorizedArrayIterator<const T>
-  begin() const
+  VectorizedArrayIterator<T>
+  end()
   {
-    return VectorizedArrayIterator<const T>(static_cast<const T &>(*this), 0);
+    return VectorizedArrayIterator<T>(static_cast<T &>(*this),
+                                      T::n_array_elements);
   }
 
   /**
-   * @return An iterator pointing to the end of the underlying data (const
+   * @return An iterator pointing to the end of the underlying data (`const`
```

Comment:
I am pretty sure, I have copied if from somewhere without the apostrophe ;)

Reply:
Oh, I'm sure there are dozens of places.